### DISCUSSION TITLE
How to set a custom region for a bucket? (I tried everything)

### DISCUSSION THREAD

**Discussion by anna-lena-strauss-converia** [minio/minio#20683] [Q&A]
I'm really struggling creating a bucket in a specified region. I tried setting the "server-location" configuration via the web UI to "eu-central-1". That worked. BUT: This morning I wasn't able to login to the web UI anymore ("The authorization header is malformed; the region is wrong; expecting 'us-east-1'."). Turned out this configuration was the issue. My hosting provider said this is a bug on minio and did a reset for me.

So, I thought maybe I can override the default ("us-east-1") while creating a bucket.

Tried it with the PHP AWS\S3Client configured to "eu-central-1". Still got "us-east-1" querying getBucketLocation.
Tried it via "mc mb --region=eu-central-1 testbucket". Same result.

What am I doing wrong?

**Comment by klauspost** [comment]
There is no per bucket region in MinIO. The region is on the server/cluster level.

**Reply by anna-lena-strauss-converia** [reply]
But how do I change this without breaking the web UI login?

**Reply by anna-lena-strauss-converia** [reply]
I want all my buckets in "eu-central-1", so I'd be happy to do this on server level.
Honestly, I don't understand the --region option. Why is it there if it isn't possible to change the region?
